### PR TITLE
Add test services

### PR DESCRIPTION
**Pull request by jenniferemshepherd**
These services are rarely used but can trigger the healthcheck (which is confusing) if they are.

See [ft-next-support ticket](https://financialtimes.atlassian.net/secure/RapidBoard.jspa?rapidView=1137&projectKey=NOPS&modal=detail&selectedIssue=NOPS-741)

### CODE REVIEW

File: lib/metrics/services.js [hunk 1]
```diff
@@ -2,13 +2,14 @@ module.exports = {
 	'acast': /^https:\/\/media\.acast\.com/,
 	'acast-thumbnails': /^https:\/\/thumborcdn\.acast\.com/,
 	'access': /^https:\/\/access\.ft\.com/,
-	'access-licence-svc': /^https:\/\/(beta-)?api\.ft\.com\/licences\//,
+	'access-licence-svc': /^https:\/\/(beta-)?api\.ft\.com\/licences\/?,
```
Comment (apaleslimghost) [inline lib/metrics/services.js:5]: this is missing the trailing `/` to close the regex:

```suggestion
	'access-licence-svc': /^https:\/\/(beta-)?api\.ft\.com\/licences\/?/,
```

File: lib/metrics/services.js [hunk 2]
```diff
@@ -2,13 +2,14 @@ module.exports = {
...
 	'access-licence-svc-RETIRED': /^https:\/\/acc-licence-svc-gw-eu-west-1-prod\.memb\.ft\.com\//,
 	'access-licence-svc-RETIRED-test': /^https:\/\/acc-licence-svc-gw-eu-west-1-test\.memb\.ft\.com\//,
-	'access-licence-svc-test': /^https:\/\/(beta-)?api-t\.ft\.com\/licences\//,
+	'access-licence-svc-test': /^https:\/\/(beta-)?api-t\.ft\.com\/licences\/?,
```
Comment (apaleslimghost) [inline lib/metrics/services.js:8]:
```suggestion
	'access-licence-svc-test': /^https:\/\/(beta-)?api-t\.ft\.com\/licences\/?/,
```

**Review by apaleslimghost — COMMENTED** [review]
there's couple of syntax errors, but the rest looks good

**Comment by jenniferemshepherd** [comment]
You reviewed this so fast @apaleslimghost! I wasn't expecting that on a Thursday afternoon 👀  sorry - added these changes in the same commit and pushed in secret....

**Review by apaleslimghost — APPROVED** [review]

**Comment by apaleslimghost** [comment]
i was looking for something to do to get me to 5pm...

**Pull request merged**